cityofaustin/vision-zero
# Move layers into `geo` schema

## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/19415

## Testing

**URL to test:** Local

We just want to make sure that VZ is business as usual.

1. Start your local stack, and apply migrations and metadata
2. Run the CRIS import, something like this should do the trick:

```
 docker run -it  --rm --env-file local.env --network host -v $PWD:/app cris_import_cris_import ./cris_import.py --csv --skip-unzip
```

3. Follow the [load_agol_layer tool instructions ](https://github.com/cityofaustin/vision-zero/tree/more-layer-updates/toolbox/load_agol_layer) to load a layer or two into your local instance

4. Start up the VZE. Edit a crash's location. Make sure the locations pages load.
5. In the VZE, test the upload non-cr3 tool


---

#### Ship list
